package.json

```diff
@@ -11,7 +11,11 @@
     "type": "git",
     "url": "git+https://github.com/Charleston-Blockchain-Devs/attendance_incentivizer.git"
   },
-  "keywords": ["Ethereum", "Solidity", "Truffle"],
+  "keywords": [
```

Comment:
keep short arrays, on single line (adjust and IDE auto fix settings to prevent this from reoccuring)